benchmark/src/main/java/ai/timefold/solver/benchmark/impl/result/ProblemBenchmarkResult.java

```diff
@@ -419,45 +420,42 @@ private void determineWinningScoreDifference() {
      * HACK to avoid loading the problem just to extract its problemScale.
      * Called multiple times, for every {@link SingleBenchmarkResult} of this {@link ProblemBenchmarkResult}.
      *
-     * @param registeringEntityCount {@code >= 0}
-     * @param registeringVariableCount {@code >= 0}
-     * @param registeringProblemScale {@code >= 0}
+     * @param problemStatistics never null
      */
-    public void registerScale(long registeringEntityCount, long registeringVariableCount,
-            long registeringMaximumValueCount, long registeringProblemScale) {
+    public void registerScale(ProblemSizeStatistics problemStatistics) {
```

Comment:
Arguably `problemSizeStatistics` now. 
Please looks for all uses of `problemStatistics` (incl. in getter/setter names) and make it consistent - looking at the PR, there are many places where you kept the old name, and I won't name all of them.